joschahenningsen/TUM-Live
# Show public courses if user courses are empty

### Motivation and Context
Resolves issue described in #1038.

### Description
Use public courses instead of user courses for **`Recent VODs`** if `usercourses.length == 0`.

### Steps for Testing


1.1 Don't log in
1.2 Navigate to `/new`
1.3 Public courses should be shown as **`Recent VODs`**

2.1 Log in 
2.2 Navigate to `/new`
3.3 **`Recent VODs`** still work for user courses
